src/rdkafka.c

```diff
@@ -270,7 +270,51 @@ void rd_kafka_log0 (const rd_kafka_conf_t *conf,
         rd_kafka_log_buf(conf, rk, level, fac, buf);
 }
 
+void rd_kafka_oauthbearer_token_refresh_success(rd_kafka_t *rk,
+                const char *token_value, int64_t md_lifetime_ms,
```

Comment:
JSON Web Tokens express their lifetime via the `exp` claim, which is an absolute time expressed as the number of (possibly fractional) seconds since the epoch (https://tools.ietf.org/html/rfc7519#section-4.1.4).  JWTs are of course not the only type of token, but they demonstrate the need to support fractional seconds of lifetime.  The Java client stores the token lifetime metadata as the number of milliseconds since the epoch in order to support this, and I chose the same metadata format here.  Unless I am misunderstanding something (possible), I will assume this choice is appropriate/acceptable.